edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/agentStart.sh

```diff
@@ -34,7 +34,7 @@ then
   if ! getent passwd "${TARGET_UID}" >/dev/null
   then
     echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Creating UID ${TARGET_UID} as edgeagentuser"
-    adduser -D -S -H -s /sbin/nologin -u "${TARGET_UID}" edgeagentuser
+    adduser -D -S -H -s /bin/sh -u "${TARGET_UID}" edgeagentuser
```

Comment:
why this change? Are we sure this won't regress anything?

Reply:
This change is just restoring the original value. If you look at the full PR diff, there's no change.